api/tests/unit/features/workflows/core/test_unit_workflows_models.py

```diff
@@ -76,6 +76,18 @@ def test_change_request_approve_by_new_approver_when_approvals_exist(
     ).exists()
 
 
+def test_change_request_is_approved_returns_true_when_minimum_change_request_approvals_is_none(
+    change_request_no_required_approvals, mocker, environment
+):
+    # Given
+    change_request_no_required_approvals.environment.minimum_change_request_approvals = (
+        None
+    )
+    change_request_no_required_approvals.save()
+    # Then
+    change_request_no_required_approvals.is_approved() is True
```

Comment:
I don't _think_ this is doing anything? I think you need you need the `assert` as per: 

```suggestion
    assert change_request_no_required_approvals.is_approved() is True
```